clients/javascript_client/README.md

```diff
@@ -1 +1,12 @@
 # Javascript Client for FMA service
+An api client wrapper for connecting to the federated-learning-service.
+
+## Setup
+Please run `npm setup` at `clients/javascript_client` directory
+
+### Example Setup
+[README.md](examples/client_examples/javascript_client/README.md)
```

Comment:
maybe render the other README here... so the user doesn't have to open another window

Reply:
ah I see